src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -488,3 +497,27 @@ def _get_list_runs_message(base_message, task_name=None, image=None):
     if image:
         base_message = "{} for image '{}'".format(base_message, image)
     return "{}.".format(base_message)
+
+
+def _get_trigger_event_list(source_triggers,
```

Comment:
nit: _update_trigger_event_list

Reply:
The event list is not always updated. If a user calls task update for reasons other than changing the status of ```commit-trigger-enabled``` or ```pull-request-trigger-enabled``` then the event list is not updated, but retrieved as is from server.

Reply:
Fair enough, I was under the impression that this one can also be called by the create method.